test/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectLoginTest.kt

```diff
@@ -76,8 +78,11 @@ class RedirectLoginTest : StringSpec() {
                     .`when`()
                     .get(Common.getSingleSignOnLocation(SamlProtocol.REDIRECT_BINDING))
 
+            // Get response from plugin portion
             val idpResponse = getServiceProvider(IdpResponder::class.java).getIdpRedirectResponse(response)
-            verifyResponse(idpResponse, true)
+            idpResponse.isRelayStateGiven = true
```

Comment:
✏️ Optionally:
```kotlin
            val idpResponse = getServiceProvider(IdpResponder::class.java).getIdpRedirectResponse(response).apply {
                isRelayStateGiven = true
            }
```

Reply:
I'd be curious to know what the "best practice" is here for kotlin. Or even what the preference is for experienced kotlin developers. My initial reaction is that the apply is overkill with just one property, but I know that I'm biased towards the Java way of doing things 😄

Reply:
I like that it helps to condition developers into seeing things as more final. Clearly in this case it is a mutable object, but by favoring this idiom we get into a habit of thinking of instances as things we create once and never change.

Reply:
That's a great answer.